src/patternfly/demos/Icons/styles.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../patternfly-utilities";
+
+// Component variables follow this formula:
+// --pf-d-icons__element--modifier--state--PropertyCamelCase
+
+// The value of component scoped variables is always defined by a global variable:
+:root {
+  // Success icon color
+  --pf-icon--success__icon--Color: var(--pf-global--success-color--100);
+
+  // Warning icon color
+  --pf-icon--warning__icon--Color: var(--pf-global--warning-color--100);
+
+  // Error icon color
+  --pf-icon--error__icon--Color: var(--pf-global--danger-color--100);
+}
+
+.pf-c-icon {
+  &.pf-m-success {
+    color: var(--pf-icon--success__icon--Color);
+  }
```

Comment:
@andresgalante should confirm this, because I'm really fuzzy on these details, but should icons be added as a component instead of a demo? I remember him saying that demos should never introduce css.

Reply:
I agree with @jgiardino. If we are introducing styles for icons then it needs to be a component.